src/lib/IDE/Intellij.py

```diff
@@ -69,85 +74,60 @@ def open_mta_perspective(self):
             # Click on the MTA tab in left sidebar
             self.click_element(locator_type="image", locator="mta_tab.png")
 
-    def run_simple_analysis(self, project, migration_target, packages=[]):
-        """
-        Runs analysis by adding the project and/or packages passed as argument
+    def setup_configuration(
```

Comment:
@midays , It would be nice if this could be a fixture. Also, could this be renamed to setup_analysis_configuration ?

Reply:
You're right, but this is planned for future improvements, with it's own PR

Reply:
Sounds good. For clarity, could you rename it to setup_analysis_configuration ?

Reply:
done